controllers/reports.js

```diff
@@ -0,0 +1,142 @@
+import models from '../models';
+
+const { User, Report, article } = models;
+
+const reporting = {
```

Comment:
@niyobobo You can change this naming on reporting as it seems like continuous action.

Reply:
@shemaeric changes applied